charts/_config.yml

```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-slate
```

Comment:
I don't know that this config will be used. We might want to consider its removal, if it isn't.

Reply:
The theme used for the chart web UI.

Reply:
I think we can remove this.